ansible/roles/riakcs/configure/tasks/main.yml

```diff
@@ -68,16 +61,6 @@
   notify: restart {{ name }}
   when: _force_riak_config or not _conf.stat.exists
 
-- name: Set {{ name }} extra conf
-  template: >
-    src={{ extra_conf }}.j2
-    dest={{ _extra_conf.stat.path }}
-    owner=root
-    group=riak
-    mode=0640
-  notify: restart {{ name }}
-  when: extra_conf != "" and (_force_riak_config or not _extra_conf.stat.exists)
-
 - name: Drop {{ name }} {{ drop_conf }}
```

Comment:
This drop extra conf task snuck into an earlier PR by accident. It should have been added in this PR.